packages/exo/index.js

```diff
@@ -1,3 +1,6 @@
 export * from './src/exo-makers.js';
 
+// eslint-disable-next-line import/export -- ESLint not aware of type exports in types.d.ts
```

Comment:
That’s unfortunate and might explain some of my experience.

Reply:
yeah. I think we should consider disabling the rule. I think it has more false positives and fewer true positives than TS